src/components/detailsPage/pages/resources/ResourcesDetailsPage.tsx

```diff
@@ -0,0 +1,74 @@
+import MapCard from 'components/cards/MapCard';
+import SidebarCard from 'components/cards/SidebarCard';
+import { AuthContext } from 'context/AuthContextProvider';
+import { useContext, useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+import { CypherQueryApiService as cypherQuery } from 'services/cypherQueryApiService';
+
+import car from '../../../../assets/car.gif';
+
+import styles from './ResourcesDatails.module.css';
+import ResourcesMainContent from './ResourcesMainContent';
+
+export default function ResourcesDetailsPage() {
+  const authContext = useContext(AuthContext);
+  const [selfDescriptionData, setSelfDescriptionData] = useState(null);
+  const { resourceId } = useParams();
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    const fetchAndSetSelfDescriptions = async () => {
+      try {
+        const response = await cypherQuery.getOneSelfDescriptions(resourceId);
+        console.log('Fetched data: ', response);
+        if (response) {
+          setSelfDescriptionData(response);
+        }
+      } catch (error) {
+        console.error('Error fetching self descriptions:', error);
```

Comment:
The error is already logged inside the `cypherQuery` you do not needed to do it again.